browser/metrics/metrics_reporting_util.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/metrics/metrics_reporting_util.h"
+
+#include "base/logging.h"
+#include "chrome/common/channel_info.h"
+#include "components/version_info/channel.h"
+
+bool GetDefaultPrefValueForMetricsReporting() {
+  switch (chrome::GetChannel()) {
+    case version_info::Channel::STABLE:  // fall through
+    case version_info::Channel::BETA:
+      return false;
```

Comment:
Please adjust this to have BETA true, unless you want it in a different PR.
Also we should wait to merge this for @jumde 's merge.

Reply:
@bbondy that is handled here:
https://github.com/brave/brave-core/pull/2042

Reply:
Yep, I just wanted to use same commit with other branch.